tests/kafkatest/services/templates/mirror_maker_consumer.properties

```diff
@@ -16,9 +16,11 @@
 
 {% if new_consumer %}
 bootstrap.servers={{ source.bootstrap_servers(security_config.security_protocol) }}
+auto.offset.reset={{ source_auto_offset_reset|default('latest') }}
 {% else %}
 zookeeper.connect={{ source.zk.connect_setting() }}
 zookeeper.connection.timeout.ms={{ zookeeper_connection_timeout_ms|default(6000) }}
+auto.offset.reset={{ source_auto_offset_reset|default('largest') }}
```

Comment:
Uggh, I forgot this about jinja2 -- `default` only applies if the value is not defined, whereas `None` counts as a value. That is why elsewhere we have `if` statements like `if x is defined and x is not none`. So these are actually getting `None` instead of the default with the current code.

Not sure which we want to prefer here. The options are either take advantage of `default` and get concision in the template but leave things undocumented in the class (i.e. to use the flag, you need to know to go look at the template files); or use the the more verbose if statements and wrap them. There are also some other possibilities suggested [here](http://stackoverflow.com/questions/19614027/jinja2-template-variable-if-none-object-set-a-default-value), only doing the inline `if`/`else` with explicit check for `None` seems to work in all cases afaict...